functions-and-scope.js

```diff
@@ -16,6 +17,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+//Checken welke waardes in de array hoger dan 8 zijn
+// Een forloop schrijven en daarna een if statement
+
+let output = 0;
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+        output = output + 1;
```

Comment:
Je zou dit iets kunnen inkorten:
```suggestion
        output++;
```